lib/charms/tempo_k8s/v2/tracing.py

```diff
@@ -75,7 +75,6 @@ def __init__(self, *args):
     TYPE_CHECKING,
     Any,
     Dict,
-    Iterable,
     List,
```

Comment:
Nit: should we bump lib versions for this or just assume it will get released with the next one?

Reply:
oh, I thought I'd rolled that back. Local tox -e lint complained about it.